app/api/__init__.py

```diff
@@ -463,11 +463,11 @@
           '/feedbacks/<int:id>/relationships/user')
 
 # Stripe Authorization API
-api.route(StripeAuthorizationListPost, 'stripe_authorization_list_post', '/stripe-authorization')
-api.route(StripeAuthorizationDetail, 'stripe_authorization_detail',  '/stripe-authorization/<int:id>',
-          '/events/<int:event_id>/stripe-authorization', '/events/<event_identifier>/stripe-authorization')
+api.route(StripeAuthorizationListPost, 'stripe_authorization_list_post', '/stripe-authorizations')
+api.route(StripeAuthorizationDetail, 'stripe_authorization_detail',  '/stripe-authorizations/<int:id>',
+          '/events/<int:event_id>/stripe-authorizations', '/events/<event_identifier>/stripe-authorizations')
```

Comment:
Please don't change the url for the event relationship.
It should still be `/events/<int:event_id>/stripe-authorization`. You can use event copyright as a reference :smile: